### PR TITLE
Feature 2345/update pulse response object

### PR DESCRIPTION
**Pull request by S78901**
In relation to:
https://github.com/objectcomputing/check-ins/issues/2345

### CODE REVIEW

**Review by mjperry91 — APPROVED** [review]

**Comment by S78901** [comment]
@mkimberlin Please give this an eyeball/approval before I merge.

File: server/src/main/java/com/objectcomputing/checkins/services/pulseresponse/PulseResponse.java [hunk 1]
```diff
@@ -141,16 +160,17 @@ public boolean equals(Object o) {
     public String toString() {
         return "PulseResponse{" +
                 "id=" + id +
+                ", internal_score" + internalScore +
+                ", external_score" + externalScore +
```
Comment (mkimberlin) [inline server/src/main/java/com/objectcomputing/checkins/services/pulseresponse/PulseResponse.java:164]: `internalScore` and `externalScore` ...these are the Java fields, not the db columns.

File: server/src/main/java/com/objectcomputing/checkins/services/pulseresponse/PulseResponseCreateDTO.java [hunk 2]
```diff
@@ -11,12 +11,16 @@
...
     @NotNull
-    @Schema(required = true, description = "date for submissionDate")
-    private LocalDate submissionDate;
+    @Schema(required = true, description = "integer value of internal score")
+    private Integer internalScore;
 
     @NotNull
-    @Schema(required = true, description = "date for updatedDate")
-    private LocalDate updatedDate;
+    @Schema(required = true, description = "integer value of external score")
```
Comment (mkimberlin) [inline server/src/main/java/com/objectcomputing/checkins/services/pulseresponse/PulseResponseCreateDTO.java:18]: I think we should make external scores and feelings nullable. That would mean changing the field below this as well. We definitely want both feelings to be nullable. Those are comments. The only thing that I think we want to require is a score on the "internal" (about work) feelings.

File: server/src/main/java/com/objectcomputing/checkins/services/pulseresponse/PulseResponse.java (hunk 1 above)
Comment (S78901) [reply server/src/main/java/com/objectcomputing/checkins/services/pulseresponse/PulseResponse.java]: Adjusted

File: server/src/main/java/com/objectcomputing/checkins/services/pulseresponse/PulseResponseCreateDTO.java (hunk 2 above)
Comment (S78901) [reply server/src/main/java/com/objectcomputing/checkins/services/pulseresponse/PulseResponseCreateDTO.java:19]: Adjusted + tests

**Comment by S78901** [comment]
@mkimberlin please review the changes

File: server/src/main/java/com/objectcomputing/checkins/services/pulseresponse/PulseResponseCreateDTO.java (hunk 2 above)
Comment (mkimberlin) [reply server/src/main/java/com/objectcomputing/checkins/services/pulseresponse/PulseResponseCreateDTO.java:19]: Almost what I meant... There are three fields that should be nullable: internalFeelings, externalScore, and externalFeelings

internalScore should not be nullable.

Comment (S78901) [reply server/src/main/java/com/objectcomputing/checkins/services/pulseresponse/PulseResponseCreateDTO.java]: @mkimberlin This has been done.

**Pull request merged**